pkg/fixtures/image.go

```diff
@@ -3328,19 +3328,19 @@ var (
 		{"quay.io/coreos/etcd:v2.2.3", "sha256:40e7261c3b72b2a4ef6ae536d316caa53203a1588d732b55b6c98731b9ba681c"},
 		{"quay.io/coreos/etcd:v2.2.4", "sha256:d0816485eaee28d7017fc744f4e64fd0208aa24d9569cf36c7cf309396eaf045"},
 		{"quay.io/coreos/etcd:v2.2.5", "sha256:f42612de899348ba8f0e96120464b2062ea53fd4199343964f602b702c65a3b4"},
-		{"quay.io/openshift/community-e2e-images:e2e-31-k8s-gcr-io-prometheus-to-sd-v0-5-0-6JI59Yih4oaj3oQOjRfhyQ", "sha256:14666989f40bb7c896c3e775a93c6873e2b791d65bc65579f58a078b7f9a764e"},
-		{"quay.io/openshift/community-e2e-images:e2e-36-k8s-gcr-io-sd-dummy-exporter-v0-2-0-pPp4DDTpVamU2YNiNfTgVQ", "sha256:6057499be1ceca52655553ff2d9361d88580da2edd5fb1c5d4cff1156ae0fdaa"},
-		{"quay.io/openshift/community-e2e-images:e2e-30-k8s-gcr-io-prometheus-dummy-exporter-v0-1-0-fNZIUrKnAwcvKTvFkmEBKw", "sha256:dfd3c8dbb1caec41ca21fe6c77138cbf730b2481987793aaefccbf8b2e80f803"},
+		{"quay.io/openshift/community-e2e-images:e2e-31-registry.k8s.io-prometheus-to-sd-v0-5-0-6JI59Yih4oaj3oQOjRfhyQ", "sha256:14666989f40bb7c896c3e775a93c6873e2b791d65bc65579f58a078b7f9a764e"},
+		{"quay.io/openshift/community-e2e-images:e2e-36-registry.k8s.io-sd-dummy-exporter-v0-2-0-pPp4DDTpVamU2YNiNfTgVQ", "sha256:6057499be1ceca52655553ff2d9361d88580da2edd5fb1c5d4cff1156ae0fdaa"},
+		{"quay.io/openshift/community-e2e-images:e2e-30-registry.k8s.io-prometheus-dummy-exporter-v0-1-0-fNZIUrKnAwcvKTvFkmEBKw", "sha256:dfd3c8dbb1caec41ca21fe6c77138cbf730b2481987793aaefccbf8b2e80f803"},
 		{"quay.io/openshift/community-e2e-images:e2e-33-gcr-io-kubernetes-e2e-test-images-regression-issue-74839-amd64-1-0-zKI5eDEBSTF6x1_LICVZ9w", "sha256:3b36bd80b97c532a774e7f6246797b8575d97037982f353476c703ba6686c75c"},
```

Comment:
This should not be changed

Reply:
Oops, should have used `[.]` rather than `.` in the regex.
But WAT? Why didn't this cause any failures? :thinking:

Reply:
This is used for scale testing, so it's possible those images were not used or the errors were ignored. Let's switch them back

Reply:
Done.